README.md

```diff
@@ -36,3 +36,4 @@ Main functions of the tool:
     ```
      ./start.sh
     ```
+    NOTE: If you come across the error `No such file or directory: '/<MARKDOWN_SOURCE_FILE_HOME>/mdsource/htmldest/'`, try executing the `easy_install beautifulsoup4` command.
```

Comment:
The error is actually not this. It is an error that says cannot find module bs4 or similar.  Can you change it to the actual error message please?

Reply:
Traceback (most recent call last):
  File "pre_processing.py", line 2, in <module>
    from bs4 import BeautifulSoup
ImportError: No module named bs4